release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1247,6 +1247,10 @@ If you are using the option `--keep-manifest-list=true`, the only valid value fo
 
 * With Serverless 0.10, the latest supported versions were updated for `trigger`, `subscription`, `channel`, and `IMC`.  Static models corresponding to each showed an API version of `beta`. The API version for eventing resources is now updated to `v1` and the UI now shows the latest supported version. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1890104[*BZ#1890104*])
 
+* Previously, when the user switched between workloads on the *Monitoring Dashboard* tab, for example from deployment to *All workloads*, the dashboard displayed a white canvas and no chart was displayed. This issue has been fixed now and the dashboard displays charts when the user switches between workloads. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1911129[*BZ#1911129*])
```

Comment:
for example, from a specific deployment to *All workloads*,

Reply:
Done!